src-tauri/src/platform_specific/linux.rs

```diff
@@ -0,0 +1,16 @@
+// Linux specific code
+
+use std::process::Command;
+
+pub fn check_glibc_v() -> f32 {
+    let out = Command::new("/bin/ldd")
+        .arg("--version")
+        .output()
+        .expect("failed to run 'ldd --version'");
+    
+    let lddva = String::from_utf8_lossy(&out.stdout);
+    let lddvl: Vec<&str> = lddva.split('\n').collect();
+    let lddvw: Vec<&str> = lddvl[0].split(' ').collect();
+    let lddv = &lddvw[3].to_string().parse::<f32>().unwrap();
```

Comment:
Regex might have been better but if works, idc :P

Similar with the `.expect`s and `.unwrap`s, in the sense that they should return an error type but not super important, I can always fix that later ^^

Reply:
I have no idea what's going on here, I wouldn't mind some comments :)